openshift/templates/email-verification-agent/email-verification-agent-deploy.yaml

```diff
@@ -66,50 +139,70 @@ objects:
     metadata:
       name: ${NAME}
       labels:
-        name: ${NAME}
-        app: ${APP_GROUP}
+        app: ${APP_NAME}${SUFFIX}
+        name: ${NAME}${SUFFIX}
         app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
     stringData:
       seed: ${WALLET_SEED}
       key: ${WALLET_KEY}
     type: Opaque
+  
+  - kind: Secret
+    apiVersion: v1
+    metadata:
+      name: ${NAME}-api${SUFFIX}
```

Comment:
Typically the naming convention for this sort of thing would be; `${NAME}${SUFFIX}-api`

Reply:
fixed